### PR TITLE
Profilecardedits

### PR DESCRIPTION
**Pull request by Sandesh1013**

### CODE REVIEW

File: src/components/ProfileCard.vue [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-                  v-row.ml-1
+                    h4.mt-1.mb-n2 {{ Profile.name }}
+                    v-row.justify-center
+                      v-col
+                        h6 {{ Profile.currentDesignation }} | {{Profile.devlupDesignation }}
+                  v-row(align="center" justify="center") 
                     v-btn(v-for='link in Profile.links', :key='link.name',:color='link.iconColor',:href="link.href" icon small)
                       v-icon(:color='link.iconColor') {{link.icon}}
-                v-icon(@click="pageNo = 2" color="basic") mdi-chevron-right
+                v-icon(@click="pageNo = 2" color="#1b65c4") mdi-chevron-right
```
Comment (Tanuj22) [inline src/components/ProfileCard.vue:22]: Undo this change.

Comment (Sandesh1013) [reply src/components/ProfileCard.vue:22]: Color was kept same as the footer
